examples/java/src/main/java/com/xebia/functional/xef/java/auto/PDFDocument.java

```diff
@@ -0,0 +1,53 @@
+package com.xebia.functional.xef.java.auto;
+
+import com.xebia.functional.tokenizer.ModelType;
+import com.xebia.functional.xef.textsplitters.TextSplitter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+import static com.xebia.functional.xef.textsplitters.TokenTextSplitterKt.TokenTextSplitter;
+
+public class PDFDocument {
+
+    public static class AIResponse {
+        public String answer;
+        public String source;
+    }
+
+    private static final String pdfUrl = "https://people.cs.ksu.edu/~schmidt/705a/Scala/Programming-in-Scala.pdf";
```

Comment:
Rename this constant name, all upper case, example: PDF_URL

Reply:
Fixed